op-deployer/pkg/deployer/integration_test/apply_test.go

```diff
@@ -366,13 +366,21 @@ func validateOPChainDeployment(t *testing.T, cg codeGetter, st *state.State, int
 
 		alloc := chainState.Allocs.Data.Accounts
 
-		firstChainIntent := intent.Chains[0]
-		checkImmutable(t, alloc, predeploys.BaseFeeVaultAddr, firstChainIntent.BaseFeeVaultRecipient)
-		checkImmutable(t, alloc, predeploys.L1FeeVaultAddr, firstChainIntent.L1FeeVaultRecipient)
-		checkImmutable(t, alloc, predeploys.SequencerFeeVaultAddr, firstChainIntent.SequencerFeeVaultRecipient)
-
-		require.Equal(t, int(firstChainIntent.Eip1559Denominator), 50, "EIP1559Denominator should be set")
-		require.Equal(t, int(firstChainIntent.Eip1559Elasticity), 6, "EIP1559Elasticity should be set")
+		chainIntent := intent.Chains[i]
+		checkImmutable(t, alloc, predeploys.BaseFeeVaultAddr, chainIntent.BaseFeeVaultRecipient)
+		checkImmutable(t, alloc, predeploys.L1FeeVaultAddr, chainIntent.L1FeeVaultRecipient)
+		checkImmutable(t, alloc, predeploys.SequencerFeeVaultAddr, chainIntent.SequencerFeeVaultRecipient)
+
+		// ownership slots
+		var addrAsSlot common.Hash
+		addrAsSlot.SetBytes(chainIntent.Roles.L1ProxyAdminOwner.Bytes())
+		// slot 0
+		ownerSlot := common.Hash{}
+		checkStorageSlot(t, alloc, predeploys.ProxyAdminAddr, ownerSlot, addrAsSlot)
+		checkStorageSlot(t, alloc, predeploys.GovernanceTokenAddr, ownerSlot, common.Hash{})
```

Comment:
The owner slot for the governance token appears to be slot 9, not slot 0
```solidity
contract GovernanceToken is ERC20Burnable, ERC20Votes, Ownable {
```

```
$ cast storage 0x4200000000000000000000000000000000000042 -r $OPTIMISM_RPC_URL -e $OPTIMISM_API_KEY 
No matching artifacts found, fetching source code from Etherscan...
| Name                    | Type                                               | Slot | Offset | Bytes | Value                                                                         | Hex Value                                                          | Contract                     |
|-------------------------|----------------------------------------------------|------|--------|-------|-------------------------------------------------------------------------------|--------------------------------------------------------------------|------------------------------|
| _balances               | mapping(address => uint256)                        | 0    | 0      | 32    | 0                                                                             | 0x0000000000000000000000000000000000000000000000000000000000000000 | Contract.sol:GovernanceToken |
| _allowances             | mapping(address => mapping(address => uint256))    | 1    | 0      | 32    | 0                                                                             | 0x0000000000000000000000000000000000000000000000000000000000000000 | Contract.sol:GovernanceToken |
| _totalSupply            | uint256                                            | 2    | 0      | 32    | 4294963294239972048644009441                                                  | 0x00000000000000000000000000000000000000000de0b5dab7d068cfc22aede1 | Contract.sol:GovernanceToken |
| _name                   | string                                             | 3    | 0      | 32    | 35931405354226271251204052796534122752185291524253814887595256144064371752976 | 0x4f7074696d69736d000000000000000000000000000000000000000000000010 | Contract.sol:GovernanceToken |
| _symbol                 | string                                             | 4    | 0      | 32    | 35874062803260315427859272455084217545361233042162567325701092311045351931908 | 0x4f50000000000000000000000000000000000000000000000000000000000004 | Contract.sol:GovernanceToken |
| _nonces                 | mapping(address => struct Counters.Counter)        | 5    | 0      | 32    | 0                                                                             | 0x0000000000000000000000000000000000000000000000000000000000000000 | Contract.sol:GovernanceToken |
| _delegates              | mapping(address => address)                        | 6    | 0      | 32    | 0                                                                             | 0x0000000000000000000000000000000000000000000000000000000000000000 | Contract.sol:GovernanceToken |
| _checkpoints            | mapping(address => struct ERC20Votes.Checkpoint[]) | 7    | 0      | 32    | 0                                                                             | 0x0000000000000000000000000000000000000000000000000000000000000000 | Contract.sol:GovernanceToken |
| _totalSupplyCheckpoints | struct ERC20Votes.Checkpoint[]                     | 8    | 0      | 32    | 28                                                                            | 0x000000000000000000000000000000000000000000000000000000000000001c | Contract.sol:GovernanceToken |
| _owner                  | address                                            | 9    | 0      | 20    | 526977378938449274083332880047510503593262190597                              | 0x0000000000000000000000005c4e7ba1e219e47948e6e3f55019a647ba501005 | Contract.sol:GovernanceToken |
```

Reply:
fixed, nice catch